### DISCUSSION TITLE
Why are several higher-level CBM I/O functions commented out of the documentation?

### DISCUSSION THREAD

**Discussion by elderling** [cc65/cc65#1534] [Q&A] [answered]
I was trying to do some CBM file I/O using the documentation on the [official documentation page](https://cc65.github.io/doc) and couldn't get things to work. This led me to go to the cc65 source code where I discovered several extremely useful, higher-level CBM file I/O functions. I was going to open a Github Issue about missing documentation only to discover that the functions exist in `doc/funcref.sgml` but are commented out. Why are they commented out?

**Comment by polluks** [comment, marked as answer]
Just an internal TODO list, feel free to commit some documentation :-)

**Reply by elderling** [reply]
OK, good to know!

**Comment by greg-king5** [comment]
(The function prototypes in the headers often have comments that describe the functions.)
